sites/kit.svelte.dev/src/routes/home/Hero.svelte

```diff
@@ -11,11 +11,12 @@
 			<a class="cta" href="/docs/introduction">read the docs</a>
 		</div>
 
-		<picture class="hero-image" sizes="(min-width: 480px) 800px, (min-width: 1024px) 480px, 600px">
+		<picture class="hero-image">
 			{#each Object.entries(background.sources) as [format, images]}
 				<source
+					sizes="(min-width: 768) 1440px, 960px"
```

Comment:
```diff
- sizes="(min-width: 768) 1440px, 960px"
+ sizes="(min-width: 768px) 1440px, 960px"
```
After adding the missing `px` I've observed that Chrome, Safari, and Firefox all have different behaviours for this (tested on MacOS).
1. Firefox is the only browser that correctly implements this.
* Respects the `sizes` attribute.
* Changes the image when resizing the window.
2. Safari.
* Respects the `sizes` attribute.
* Does **not** change the image when resizing the window (it only loads the correct image on initial page load).
3. Chrome
* Does **not** respect the `sizes` attribute (always loads the largest image).
* Removing the `sizes` attribute enables `srcset` to work correctly (loads the next largest image on resize).

Would be great if someone else could verify these behaviours too.

Reply:
I can not test on Safari, but it seems Firefox behaves similarly to Chrome for me. It always loads the largest if the `sizes` attribute is present.

> Removing the sizes attribute enables srcset to work correctly (loads the next largest image on resize).

I would not say "correctly", but at least something happens.
Without the `sizes` attribute, Chrome switches to the large image when the screen width is >= 588px (DPR: 2.0) while Firefox switches to the large image when the screen width is >= 481 (DPR: 2.0)